core/args.py

```diff
@@ -13,9 +11,9 @@
 
 
 def parse_args():
-    """
-    Parses the command line arguments.
-    This function may exit if the command line arguments are invalid.
+    """Parse the command line arguments.
```

Comment:
You seem to have switched from "Parses" to "Parse", is that intentional ? If so, you should also update the other comments, in order to keep a consistent commenting style.

Reply:
Yeah I try to have the first verb in the infinitive, but it's so unnatural to me that I fail sometimes. Thanks for spotting that.